libs/design-system/nav-header/src/lib/nav-header.component.html

```diff
@@ -16,7 +16,7 @@
               [icon]="card.icon"
               [tagline]="card.name"
               [description]="card.description"
-              [link]="card.link | assetUrl"
```

Comment:
That's exactly what I thought the issue would be 😄. Thanks for getting it fixed!